src/compiler/scala/tools/nsc/typechecker/RefChecks.scala

```diff
@@ -469,10 +468,8 @@ abstract class RefChecks extends Transform {
                   if (currentUnit.sourceAt(namePos) == member.decodedName)
                     runReporting.codeAction("remove empty parameter list", namePos.focusEnd.withEnd(namePos.end + 2), "", msg, expected = Some(("()", currentUnit)))
                   else Nil
-                if (currentRun.isScala3)
-                  overrideErrorWithMemberInfo(msg, action)
-                else
-                  refchecksWarning(member.pos, msg, WarningCategory.OtherNullaryOverride, action)
+                val cat = if (currentRun.isScala3) WarningCategory.Scala3Migration else WarningCategory.OtherNullaryOverride
+                refchecksWarning(member.pos, msg, cat, action)
```

Comment:
this will conflict with my PR for the other ticket, which I can't PR just now because I'm watching Heroes.